.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+# Have you read capa's CONTRIBUTING guide?
+It contains helpful information about how to contribute to capa. Check https://github.com/fireeye/capa/blob/master/.github/CONTRIBUTING.md
+
+based on https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/
+-->
+
+### Description
+
+closes # (issue)
+
+<!-- Description of the PR -->
+
+### Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+### Tests
+
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] No new tests needed
+
+### Checklist
+
+- [ ] I have updated the [CHANGELOG.md](/CHANGELOG.md)
```

Comment:
Should we set up the GitHub action and a templete in the CHANGELOG to ensure this is done before adding it here? I was thinking about setting this up together with the next release to be able to start with a empty CHANGELOG.

In https://github.com/fireeye/capa/issues/457 I suggested to use a label if no CHANGELOG changes are needed. But contributors don't have permissions for that. Maybe a section like the one for the tests would be better, so that we can add the label if no changes are needed. Something like:
```
### Changelog

- [ ] I have updated the [CHANGELOG.md](/CHANGELOG.md)
- [ ] The [CHANGELOG.md](/CHANGELOG.md) doesn't need to be updated (changelog linter will fail until a maintainer adds the `no changelog` label)
```

Reply:
> Should we set up the GitHub action and a templete in the CHANGELOG to ensure this is done before adding it here?

1. Adding the template is a good idea, I think the Action is optional for now.

2. Wouldn't any change require a respective CHANGELOG entry? Let's try to keep it simple.

Reply:
@mr-tz 

> Wouldn't any change require a respective CHANGELOG entry? Let's try to keep it simple.

We could give it a try. 🤔